telegram/bot.py

```diff
@@ -579,6 +579,14 @@ def forward_message(
     ) -> Message:
         """Use this method to forward messages of any kind. Service messages can't be forwarded.
 
+        Note:
+            Since the release of Bot API 5.5 it can be impossible to forward messages from
+            some chats. Use the field :attr:`telegram.Message.has_protected_content` and
+            :attr:`telegram.Chat.has_protected_content` to check this.
+
+            As a workaround, it is still possible to use :meth:`copy_message`. However, this
+            behaviour is undocumented and might be changed by Telegram.
```

Comment:
wut? this is stupid 🤦

Reply:
indeed, guess its not impossible after all